### DISCUSSION TITLE
What happened to MD3 based drift detectors?

### DISCUSSION THREAD

**Discussion by TzOk83** [IFCA-Advanced-Computing/frouros#280] [General]
With the release of v0.2.0 support for MD3 detectors have been removed, but with a promise of bringing them back in the future releases. Now we have v0.6.x and still no sign of them. Will they finally reappear or it is a lost case?

**Comment by jaime-cespedes-sisniega** [comment]
Hi @TzOk83,

For any version 0.X.X, there is no possibility of reintroducing MD3 detectors due to the significant modifications required at the API level. However, starting from version 1.X.X, there is an intention to bring them back.
